src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -264,6 +331,47 @@ const getMenuBorderRadius = ({blendCorner, alignMenu, theme}) => {
   `;
 };
 
+const getMenuArrow = ({menuWithArrow, alignMenu}) => {
+  if (!menuWithArrow) return '';
+  let alignRight = alignMenu === 'right';
+
+  return css`
+    top: 32px;
```

Comment:
This is C&P'ed from existing component :o

Reply:
holy moly that's a lot of stuff lol. maybe we can just transform one of our `carat` svgs?

Reply:
🙃